jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -237,19 +237,21 @@ def test_wcslink_rotated(self):
         assert not label_mouseover.row2_unreliable
         assert not label_mouseover.row3_unreliable
 
-        # Make sure GWCS now can extrapolate.
+        # Make sure GWCS does not extrapolate.
         xy = self.viewer._get_real_xy(self.imviz.app.data_collection[1], -1, -1, reverse=True)
         label_mouseover._viewer_mouse_event(
             self.viewer, {'event': 'mousemove', 'domain': {'x': xy[0], 'y': xy[1]}})
-        assert label_mouseover.as_text() == ('Pixel x=-1.0 y=-1.0',
-                                             'World 00h14m19.5829s -30d23m30.9860s (ICRS)',
-                                             '3.5815955408 -30.3919405616 (deg)')
+        assert label_mouseover.as_text() == ('Pixel', '', '')

```

Comment:
Shouldn't `'Pixel x=-1.0 y=-1.0'` still display? The mouse location is still valid, just the WCS is NaN.

Reply:
I'm not sure. If we're self-consistently _not_ showing out-of-bounds coordinates, I think that should apply in both p2w/w2p transformations.

Reply:
I still want the X/Y even when out of bounds. Can you please put that back? Thanks.